lib/rules/no-unescaped-entities.js

```diff
@@ -28,10 +40,7 @@ module.exports = {
       type: 'object',
       properties: {
         forbid: {
-          type: 'array',
-          items: {
-            type: 'string'
```

Comment:
these should be tightly specified - either a string, or an object with the proper "char" + "alternatives" schema

Reply:
Yeah, I tried. I couldn't find a `Either` type with this lib.